eng/targets/Settings.props

```diff
@@ -149,14 +148,14 @@
 
   <!-- Language-specific analyzer packages -->
   <Choose>
-    <When Condition="'$(Language)' == 'VB' and '$(RoslynCheckCodeStyle)' == 'true'">
-      <ItemGroup>
+    <When Condition="'$(Language)' == 'VB'">
+      <ItemGroup Condition="'$(RoslynCheckCodeStyle)' == 'true'">
```

Comment:
The property this is conditioning on is defined in the targets file now, not the props. Think we should move this `<Choose>` to the targets file as well.

Reply:
All property groups are evaluated before all item groups. The SDK itself has many examples of cases where default properties are assigned in the targets file (allowing a project file to set values before the defaults are applied as fallback), and items in the props file (allowing a project file to `Update` one or more items).